src/sentry/toolbar/iframe_view.py

```diff
@@ -0,0 +1,40 @@
+from typing import Any
+
+from django.http import HttpRequest, HttpResponse
+
+from sentry.web.frontend.base import OrganizationView, region_silo_view
+
+
+@region_silo_view
+class IframeView(OrganizationView):
+    # TODO: For perms check. This is just an example and we might not need it.
+    # required_scope = "org:read,org:integrations"
+
+    security_headers = {
+        "X-Frame-Options": "ALLOWALL",
+        # TODO: # Not working, seems to get overridden by a BaseView wrapper
+        # "Content-Security-Policy": "frame-ancestors http://dev.getsentry.net/ http: https:",
```

Comment:
As long as one of these works and we can render the `iframe.html` template into an `<iframe>` node that's all we need in this PR.

I know `X-Frame-Options` was working before, but now we've got `OrganizationView` as a base class (which we want to keep!) so we just need to resolve this todo with one header or the other so it keeps working.